miniscope_io/stream_daq.py

```diff
@@ -276,8 +334,12 @@ def _fpga_recv(
         if read_length is None:
             read_length = int(max(self.buffer_npix) * self.pix_depth / 8 / 16) * 16
 
+        # set up fpga devices
+        BIT_FILE = "./devices/" + self.bitstream
+        BIT_FILE = os.path.abspath(os.path.join(os.path.dirname(__file__), BIT_FILE))
```

Comment:
This limits to only using bitfiles in the same directory as this file. Lets just take a path here and throw an error if it doesnt exist. Assume that the path is correctly passed by the config, and if it isnt raise informative error.